### ISSUE TITLE
VOCR keeps launching at login even if i disable the option in the menu extra.

### ISSUE THREAD

**Issue by vlad-a-c** [chigkim/VOCR#28]
hello.
once i disabled this option and i restarted my mac the app keeps launching. not sure if it's a system thing, but i wonder if can you get a work around to take it off the login items once i disable the option.

**Comment by chigkim** [comment]
Vocr uses launch agent feature on macos.

You can simply recheck and uncheck launch on login, or simply delete 
com.chikim.vocr.plist file in ~/Library/LaunchAgents folder.

If you can't find the file, then something else is launching the VOCR on 
login.

Maybe you added VOCR to login item from system settings in the past?

**Comment by chigkim** [comment]
Hmm, that is strange. The status of the checkbox depends on whether com.chigkim.VOCR.plist exists in ~/library/launchagents/ or not.
It works correctly from my end.
Can you delete that file, reboot, and try again?
Thanks!

**Comment by chigkim** [comment]
Yes it's in ~/library/preferences/com.chikim.VOCR.plist, but the Startup doesn't use that settings though.
Also unless you run the reset feature from the latest VOCR beta build, just deleting the file won't work. I believe MacOS caches those preference files in memory.
You would need to reboot your machine.

**Comment by chigkim** [comment]
Please check what's in your login item. System settings > general > login items.
Closing this for now. Please try the latest release, and reopen if you still have this issue with the latest release.
